docs/API.Auth.InvalidateCache.md

```diff
@@ -0,0 +1,32 @@
+# Function: *Invalidate Cache* #
+
+## Description ##
+
+CM-Well manages a cache with authentication information for better performance. If you've made a change to authentication information such as creating a new user, creating a new role or changing a password, and you want the change to take immediate effect, you will need to refresh this cache. You do this by calling the **invalidate-cache** API.
```

Comment:
CM-Well manages a cache with <s>authentication</s> **authorization** information for better performance. If you've made a change to <s>authentication</s> **authorization** information such as ...